src/clinical_zeroshot_labeler/model.py

```
@@ -0,0 +1,505 @@
+    @torch.no_grad()
+    @eval_decorator
+    def generate(
+        self,
+        prompts: torch.Tensor,
+        mask: torch.Tensor | None,
+        trajectory_labeler: SequenceLabeler | None = None,
+        time_offset_years: torch.Tensor | None = None,
+        eos_tokens: list[int] | None = None,
+        temperature: float = 1.0,
+        filter_logits_fn: str | Callable = identity,
+        filter_kwargs: dict = dict(),
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)